yb-voyager/cmd/importSchema.go

```diff
@@ -46,7 +46,8 @@ var importSchemaCmd = &cobra.Command{
 
 	Run: func(cmd *cobra.Command, args []string) {
 		tconf.ImportMode = true
-		sourceDBType = ExtractMetaInfo(exportDir).SourceDBType
+		createInitConnectToMetaDBIfRequired()
+		sourceDBType = GetSourceDBTypeFromMigInfo()
```

Comment:
What if user (by mistake) runs the `import schema` command first (even before `export schema/data`)?

Reply:
yeah same holds true for other commands like `import data`, `analyze-schema`. 
Need to error out with proper error message.